internal/engine/upper.go

```diff
@@ -875,6 +875,7 @@ func handleDockerComposeLogAction(state *store.EngineState, action DockerCompose
 	ms.ResourceState = dcState.WithCurrentLog(append(dcState.CurrentLog, action.Log...))
 }
 
-func handleTiltfileLogAction(state *store.EngineState, action TiltfileLogAction) {
+func handleTiltfileLogAction(ctx context.Context, state *store.EngineState, action TiltfileLogAction) {
 	state.CurrentTiltfileBuild.Log = model.AppendLog(state.CurrentTiltfileBuild.Log, action.Log)
+	logger.Get(ctx).Infof("[TILTFILE] " + string(action.Log))
```

Comment:
Should the name of the resource in square brackets match the casing of the resource as it appears in the resource list in the UI?

Reply:
ha sure